code/modules/mob/living/simple_animal/vore/otie.dm

```diff
@@ -113,10 +114,6 @@
 /mob/living/simple_animal/otie/Found(var/atom/found_atom)
```

Comment:
And this is why you changed nothing. Gut **everything out of here** and add **only the part that Oties do differently** than normal mobs. Now do that for everything else. Use `..()` to continue the proc's base functionality, IE, what mobs normally do.

Reply:
Thing is, all of those things there are what they do differently from normal mobs from anywhere down the parent branch. And every borrowed part there is from mobs/items completely outside the branch and thus not there for ..()ing to begin with.
![kuva](https://user-images.githubusercontent.com/13697337/30874637-a2321324-a2f9-11e7-8c52-30480730497b.png)

Reply:
Like. The beepsky code breaks off from the branch at /mob/living/bot. There ain't no 'bot' in simple_animal branch.
The kitty befriending stuff breaks off at /mob/living/simple_animal/cat/fluff, which is way on another level.
And of course there is this wonderful thing that the ..() takes the entire base proc from the single tab level, and thus cannot be used for sticking one line somewhere in a function 5 functions deeper somewhere in the middle of the proc.